benchmark/C/Savina/src/micro/count/CountingBenchmarkN.lf

```diff
@@ -0,0 +1,151 @@
+/**
```

Comment:
What should we do with the two different Versions of this benchmark? I mean `CountingBenchmark.lf` and `CountingBenchmarkN.lf`. I think it would suffice to only have the cog based version. Do you agree?

Also I think we should simplify the names. Instead of `CountingBenchmark.lf` we can just call it `Counting.lf`. This is already done in the C++ benchmarks.

Reply:
I agree, it only make sense to keep the cog version around.  I just pushed a commit with the .yaml configurations updated and the renamed Counting.lf file